docs/guides/server.md

```
@@ -0,0 +1,94 @@
+  --version v0.6.0
+```
+
+## Security Features
+
+### Authentication
+
+The API server uses Kubernetes `TokenReview` for authentication.
+It supports standard OIDC JWTs (JSON Web Tokens) or Kubernetes ServiceAccount tokens.
+
+To authenticate requests, pass your token in the `Authorization` header as a Bearer token:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)